jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/connection/workflow/v2/CawsHostCommandExecutor.kt

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.gateway.connection.workflow.v2
+
+import com.intellij.remote.RemoteCredentials
+import com.jetbrains.gateway.ssh.deploy.CommandExecutionResultWithRawStreams
+import com.jetbrains.gateway.ssh.deploy.CommandExecutionStreams
+import com.jetbrains.gateway.ssh.deploy.HostCommandExecutor
+import com.jetbrains.gateway.ssh.deploy.ShellArgument
+import com.jetbrains.gateway.ssh.deploy.impl.SshCommandExecutor
+import com.jetbrains.rd.util.lifetime.Lifetime
+
+class CawsHostCommandExecutor private constructor(private val delegate: HostCommandExecutor) : HostCommandExecutor by delegate {
```

Comment:
if we don't need the hacky input stream filter, we don't need this either

Reply:
Done